pkg/cmd/controller/psalabelsyncer.go

```diff
@@ -13,19 +14,40 @@ func runPodSecurityAdmissionLabelSynchronizationController(ctx context.Context,
 		return true, err
 	}
 
-	controller, err := psalabelsyncer.NewPodSecurityAdmissionLabelSynchronizationController(
-		kubeClient.CoreV1().Namespaces(),
-		controllerCtx.KubernetesInformers.Core().V1().Namespaces(),
-		controllerCtx.KubernetesInformers.Rbac().V1(),
-		controllerCtx.KubernetesInformers.Core().V1().ServiceAccounts(),
-		controllerCtx.SecurityInformers.Security().V1().SecurityContextConstraints(),
-		controllerCtx.EventRecorder.ForComponent("podsecurity-admission-label-sync-controller"),
-	)
+	featureGates := sets.NewString(controllerCtx.OpenshiftControllerConfig.FeatureGates...)
+	switch {
+	case featureGates.Has("OpenShiftPodSecurityAdmission=true"):
+		// if explicitly off, disable
```

Comment:
This comment is slightly confusing to me. In this case we are enabling the enforcement.